test/functional/wallet_miniscript.py

```diff
@@ -208,6 +208,7 @@ def add_options(self, parser):
 
     def set_test_params(self):
         self.num_nodes = 1
+        self.extra_args = [["-acceptnonstdtxn"]]
```

Comment:
Just want to add: even if this change were a good idea, changing the wallet tests to allow non standard txs in the mempool is a terrible idea.

The point of wallet tests is to check that we actually can create and spend payments to addresses we support. If any of those addresses aren't acceptable to our own mempool by default, that's a bug, and it shouldn't be hidden in this way. If we're deliberately making some addresses unspendable with a default config despite having previously supported them, that should be addressed by explicitly removing those classes of addresses from the test cases.